### DISCUSSION TITLE
fly keys

### DISCUSSION THREAD

**Discussion by iandoug** [marlonrichert/marlonrichert#5] [General]
Hi Marlon

is Ergo Fly Keys one of your layouts?

Thanks, Ian

**Comment by marlonrichert** [comment]
Yes, it is. Why do you ask? 🙂

**Comment by iandoug** [comment]
Checking your layout... :-)

What are all those Section keys for?

Also:
checking ergo-fly-keys.en.ansi
 Error with ergo-fly-keys.en.ansi.json : ergo-fly-keys.en.ansi, no Alt key assigned 
 Error with ergo-fly-keys.en.ansi.json : ergo-fly-keys.en.ansi, no AltGr key assigned 
 Error with ergo-fly-keys.en.ansi.json : ergo-fly-keys.en.ansi, no Left Shift key assigned 

Thanks, Ian

**Reply by marlonrichert** [reply]
You can find the full layout and documentation here: https://ke-complex-modifications.pqrs.org/#ergo_fly_keys

**Reply by iandoug** [reply]
Thanks.
